lib/application/settings/build_information/build_information_bloc.dart

```diff
@@ -19,8 +19,6 @@ class BuildInformationBloc
     on<BuildInformationEvent>((event, emit) async {
       await event.when(
         fetch: () async {
-          emit(const BuildInformationState.loading());
```

Comment:
Any particular reason for removing the loading state ??

Reply:
It's also the reason the `build_information_test` and `building_information_bloc_test` are failing

Reply:
This one is tricky, it should be the initiaæ state of the BLOC

Reply:
The blocTest expects only emitted values when an event is added. So skipping the initial state.